packages/components/src/components/chat-search/Context.vue

```diff
@@ -54,13 +54,15 @@
 <script lang="ts">
 //@ts-nocheck
 import VContextItem from '@/components/chat-search/ContextItem.vue';
+import VFile from '@/components/chat/File.vue';
 import VMarkdownCodeSnippet from '@/components/chat/MarkdownCodeSnippet.vue';
 import VMermaidDiagram from '@/components/chat/MermaidDiagram.vue';
 import VButton from '@/components/Button.vue';
 
 const PinnedContextComponents = {
   'code-snippet': VMarkdownCodeSnippet,
   mermaid: VMermaidDiagram,
+  file: VFile,
```

Comment:
I'd expect this to be registered within `components` below, is this working?

Reply:
[This](https://vuejs.org/guide/components/registration) seems to suggest that components need to be registered so when they're used within a template, Vue knows how to find the implementation. `v-file` doesn't get used in this (or any other) template, which is probably why it works (which I believe you've now seen).

Doesn't seem like it would do any harm to add it to `components`, though, and that would probably bring it more in line with standard practice.